app/views/pages/home.html.erb

```diff
@@ -2,17 +2,17 @@
 <table id="table_of_link_buttons" class="link_buttons_in_table">
   <tr>
     <td>
-      <%= link_to t("families.new.title"), new_family_path(locale: I18n.locale) %>
+      <%= link_to t("families.new.title") %>
     </td>
   </tr>
   <tr>
     <td>
-      <%= link_to t("schools.new.title"), new_school_path(locale: I18n.locale) %>
+      <%= link_to t("schools.new.title") %>
```

Comment:
You still need the link, just don't need to pass the locale :)

Reply:
I can't understand....is not correct?  🤔